skunkworks_crow/src/main/res/values/strings.xml

```diff
@@ -139,8 +139,10 @@
     <string name="connect_bluetooth_title">Device to Receive</string>
     <string name="connected_bluetooth_downloading">Connected, start downloading&#8230;</string>
     <string name="connecting_title">Connecting</string>
-    <string name="bluetooth_paired">paired, may unreachable</string>
-    <string name="bluetooth_unpaired">unpaired, reachable</string>
+    <string name="scanning_title">Scanning</string>
+    <string name="scanning_msg">Scanning bluetooth devices, please wait&#8230;</string>
+    <string name="bluetooth_paired">paired</string>
```

Comment:
It could be like `Device Paired` or `Device Unpaired`.

Reply:
Sure, will fix right now.